### ISSUE TITLE
A11y_SQL Azure Data Studio_Database_Create Project from database_Screen reader: Screen reader is not announcing " Section Headings" of "Create project from database" section. ·

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/azuredatastudio#24163]
### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Aug2023; #SQL Azure Data Studio; #WCAG1.3.1; #ScreenReader; #Win32; #Benchmark; #FTP; #STP; #Element:Heading;

### Environment Details:
Application Name: Sql Azure Data studio
Version: 1.45.1
OS: Win11

### Steps to Reproduce

1.Launch the application-Azure Database Project and install extension SQL Database Projects.
2.Navigate to SQL Database Project Home Screen with heading "Projects".
3.Tab to connections select your server and connect to it.
4.Few databases will appear under your connection then right click on any database
5.Tab till "Create project from database" and hit enter.
6. Now verify that NVDA/Narrator not announcing "Section Headings" of "Create project from database" section.

### Actual Results:
Screen reader is not announcing "Section Headings" of "Create project from database" section.

**Note:**
**Similar issue observed with chrome+ NVDA.**

### Expected Results:
Screen reader should announce "Section Headings" of "Create project from database" section.

### User Impact:
Screen reader user will not be able to know "Section Headings" of "Create project from database" section. If NVDA/Narrator not announcing.

### Attachment:
[SR is not announcing the heading along with label.webm](https://github.com/dotnet/maui/assets/142479541/f755f9e6-58cd-4804-958a-f9f87f3852eb)

[sr not announcing the heading along with label NVDA.webm](https://github.com/dotnet/maui/assets/142479541/68b47ee0-3167-4cd0-916b-3e8567c203e8)

<img width="960" alt="MicrosoftTeams-image" src="https://github.com/dotnet/maui/assets/142479541/760cb452-9f19-4a45-8f63-1e758eefa27e">

**Comment by kisantia** [comment]
@PadminiGariki please test this with the updated extension I provided as soon as you can, since this has passed the SLA. Thanks!

**Comment by PadminiGariki** [comment]
verified the bug in this environment [Latest insiders build](https://github.com/microsoft/azuredatastudio#download-the-latest-azure-data-studio-release).
Here issue is Not fixed.
for reference, please find the attachment.
#Reopened; #Regressed:10-31-23;

[Issue is still repro in NVDA.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/6a652104-960c-425a-88b0-8c4b3b2981fe)

[Issue is not fixed in Narrator.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/5ae6212c-af5b-4b85-971e-9afb42224157)

**Comment by kisantia** [comment]
@PadminiGariki that is not the latest version of the sql projects extension. Please make sure version 1.4.1 is installed as this is the version with the fix.
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/dbb40a9f-5c8a-4b8c-8824-63b0972a051c)

The section headers are no longer bolded in the latest version of the extension, 
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/74c82369-9c62-4485-8688-4a57780dfa5d)


Screenreader announcing the section heading when the first item in the group is navigated to:

https://github.com/microsoft/azuredatastudio/assets/31145923/1ca2df11-09bb-4837-8eab-44e84b5fd61f

**Comment by PadminiGariki** [comment]
@kisantia
verified the bug in this environment [Latest insiders build](https://github.com/microsoft/azuredatastudio#download-the-latest-azure-data-studio-release).
Here issue is Partially fixed. Here issue is fixed with NVDA but issue still repro in narrator.
for reference, please find the attachment.
#Reopened; #Regressed:11-09-23;

[Narrator.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/daea93e7-0213-468b-a2af-a4c06c62f1f7)

[NVDA.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/520ded2f-5214-4fe2-ab98-14017b5a1a79)

**Comment by kisantia** [comment]
@PadminiGariki I thought as long as one screen reader on a platform is reading it correctly, then an issue can be resolved.

**Comment by kisantia** [comment]
@PadminiGariki confirming that this is a Narrator issue. NVDA announces the section headings.

**Comment by PadminiGariki** [comment]
#External:Narrator; # External; #ED:PD:Narrator; #Regressed:11-21-23;

**Comment by 12-shweta** [comment]
#ED:Ack:Wipro;

**Comment by PadminiGariki** [comment]
#ED:NoRepro; #Regressed:12-14-23;

@kisantia 
This issue not only repro with Narrator. It is also repro with jaws.
Could you please verify the bug. For the reference attachment is added below.
[issue repro with Jaws.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/3382a63b-cd48-4a65-948a-4d921d0cb31d)
